Protecs/src/main/java/uk/ac/ucl/protecs/behaviours/Demography.java

```diff
@@ -0,0 +1,221 @@
+package uk.ac.ucl.protecs.behaviours;
+
+import java.util.Arrays;
+import java.util.List;
+
+import sim.engine.SimState;
+import sim.engine.Steppable;
+import uk.ac.ucl.protecs.objects.Household;
+import uk.ac.ucl.protecs.objects.Location;
+import uk.ac.ucl.protecs.objects.Person;
+import uk.ac.ucl.protecs.sim.Params;
+import uk.ac.ucl.protecs.sim.WorldBankCovid19Sim;
+
```

Comment:
keeping them as classes allows us to easily modularise the behaviours (and omit them where desired)